tuf/utils/x509.go

```diff
@@ -443,6 +454,9 @@ func EncryptPrivateKey(key data.PrivateKey, role, passphrase string) ([]byte, er
 		return nil, fmt.Errorf("unable to encrypt key - invalid PEM file produced")
 	}
 	encryptedPEMBlock.Headers["role"] = role
+	if gun != "" {
```

Comment:
@endophage IIRC previously we could import a delegation key as any role, so it can be "user" but it could by anything (our docs recommend user).  You couldn't really import a key such that it had no role header.